.github/workflows/slack-office-hours-design.yml

```diff
@@ -0,0 +1,36 @@
+name: Slack Office Hours Reminder
+
+on:
+  schedule:
+    # Run every Tuesday at 09:00AM CT, 02:00PM UTC
+    - cron: '0 14 * * WED'
+
+jobs:
+  post-reminder:
+    name: Post design reminder to carbon-design-system and carbon-announcements
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send custom JSON data to Slack workflow
+        id: slack
+        uses: slackapi/slack-github-action@v1.22.0
+        with:
+          payload: |
+            {
+              "username": "Carbon Design System",
+              "icon_url": "https://user-images.githubusercontent.com/3360588/192045905-5d9705af-92e2-4432-805e-15db98571e8b.png",
+              "channel": ["#carbon-design-system", "#carbon-announcements"],
+              "text": "Join us for Carbon Design Office Hours!",
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "Join us for Carbon Design Office Hours!",
+                    "emoji": true
+                  }
+                }
+              ]
```

Comment:
This bit here is what I can update with the contents from the slack block kit builder.